lib/src/golden_test_adapter.dart

```diff
@@ -263,11 +263,8 @@ class FlutterGoldenTestAdapter extends GoldenTestAdapter {
                       maxWidth: constraints.maxWidth,
                       maxHeight: constraints.maxHeight,
                       child: Center(
-                        child: Padding(
```

Comment:
@btrautmann what is the reasoning behind this removal of the `Padding`?

Reply:
![Screenshot 2024-10-03 at 09 17 57](https://github.com/user-attachments/assets/fa97c14d-86fe-46b8-8197-b4d802d18286)

See https://github.com/Betterment/alchemist/issues/105

I didn't think it would make sense to maintain a configuration option so I removed the padding.